internal/lib/dynamic_client.go

```diff
@@ -478,6 +478,21 @@ func WaitForInitSecretRecreateAndReboot() {
 	}
 }
 
+func GetNSToSEGMap() (map[string]string, error) {
```

Comment:
We can move it to lib/utils.go as we are not using dynamic client in this method